llvm/lib/CodeGen/AsmPrinter/DwarfDebug.cpp

```diff
@@ -1326,6 +1378,11 @@ void DwarfDebug::collectVariableInfoFromMFTable(
     auto RegVar = std::make_unique<DbgVariable>(
                     cast<DILocalVariable>(Var.first), Var.second);
     RegVar->initializeMMI(VI.Expr, VI.Slot);
+    if (VariableInDependentType.count(VI.Var)) {
+      const DIType *DT = VariableInDependentType[VI.Var];
+      if (const DIFortranArrayType *AT = dyn_cast<DIFortranArrayType>(DT))
+        constructSubrangeDie(AT, *RegVar.get(), TheCU);
+    } 
```

Comment:
Nit: trailing space on this line
```suggestion
    }
```

Reply:
Fixed.